### DISCUSSION TITLE
load/save samples

### DISCUSSION THREAD

**Discussion by heeplr** [OpenHantek/OpenHantek6022#186] [Ideas]
**Is your feature request related to a problem? Please describe.**
Inspecting signals is quite nice, but there is no way to inspect past signals. It's already possible to export to csv but no way to import.

**Describe the solution you'd like**
Menu options:
* **File -> Save** - will save the current buffer to a file.
* **File -> Load** - will load a buffer from a file and adjust the settings accordingly (samplerate etc.)

**Describe alternatives you've considered**
An alternative would be an "Import CSV" option but I think there's no need for the ASCII conversion.

**Additional context**
Thanks a lot for your work on OpenHantek6022!

**Comment by zen67** [comment]
I can test it.
